tei/la-dama-duende.xml

```diff
@@ -2320,6 +2323,8 @@
             <l n="1120">El papel, Beatriz, es este.</l>
             <l>Holgareme que te agrade.</l>
             <stage>Lee ÁNGELA.</stage>
+            <p>Fermosa dueña, cualquiera que vós seáis la condolida deste afanado caballero, y a saz piadosa minoráis sus cuitas, ruego vós me queráis facer sabidor del follón mezquino, o pagano malandrín, que en este encanto vos amancilla, para que segunda vegada en vueso nombre, sano ya de las pasadas feridas, entre en descomunal batalla, maguer que finque en ella, que non es la vida de más proo que la muerte, tenudo a su deber un caballero. El Dador de la Luz vos mampare, e a mí non olvide.</p>
```

Comment:
Maybe someone with a better grasp at normalizing the orthography of Calderón's Spanish should check this version.